packages/agoric-cli/src/lib/rpc.js

```diff
@@ -143,7 +147,11 @@ export const makeVStorage = (powers, config = networkConfig) => {
           ));
           // console.debug('readAt returned', { blockHeight });
         } catch (err) {
-          if (err.message.match(/unknown request/)) {
+          if (
+            err.message.match(/unknown request/) ||
+            // ErrNotFound CosmosSDK error
+            (err.code === 38 && err.codespace === 'sdk')
+          ) {
```

Comment:
We should probably explain a little more here.
```suggestion
          if (
            // CosmosSDK ErrNotFound; there is no data at the path
            (err.codespace === 'sdk' && err.code === 38) ||
            // CosmosSDK ErrUnknownRequest; misrepresentation of the same until
            // https://github.com/Agoric/agoric-sdk/commit/dafc7c1708977aaa55e245dc09a73859cf1df192
            // TODO remove after upgrade-12
            err.message.match(/unknown request/)
          ) {
```